doc/guide/Makefile-guide.am

```diff
@@ -98,6 +98,11 @@ dist/guide/html/index.html: $(GUIDE_DOCBOOK) $(GUIDE_INCLUDES) $(man_MANS) $(GUI
 		LANG=en_US.UTF-8 $(XMLTO) html -m $(srcdir)/doc/guide/gtk-doc.xsl -o dist/guide/html/ \
```

Comment:
Yes, I saw -- what I meant is that while your are at touching these bits it might be nice to make a separate commit which removes the existing `&&` and the extra indentation too.